src/pykeen/nn/modules.py

```diff
@@ -1835,20 +1836,30 @@ def forward(
 
 
 @parse_docdata
-class CrossEInteraction(FunctionalInteraction[FloatTensor, tuple[FloatTensor, FloatTensor], FloatTensor]):
-    """A module wrapper for the CrossE interaction function.
+class CrossEInteraction(Interaction[FloatTensor, tuple[FloatTensor, FloatTensor], FloatTensor]):
+    r"""The CrossE interaction function.
 
-    .. seealso:: :func:`pykeen.nn.functional.cross_e_interaction`
+    .. math ::
+        Dropout(Activation(c_r \odot h + c_r \odot h \odot r + b))^T t)
+
+    .. note ::
+        The CrossE paper described an additional sigmoid activation as part of the interaction function. Since using a
+        log-likelihood loss can cause numerical problems (due to explicitly calling sigmoid before log), we do not use
+        it in our implementation, but opt for the numerically stable variant. However, the model itself has an option
+        ``predict_with_sigmoid``, which can be used to force the use of sigmoid during inference. This can also affect
+        rank-based scoring, since limited numerical precision can lead to exactly equal scores for multiple choices.
+        The definition of a rank is not unambiguous in this case, and there are several competing ways to break ties.
+        See :ref:`understanding-evaluation` for more information.
 
     ---
     citation:
         author: Zhang
         year: 2019
         link: https://arxiv.org/abs/1903.04750
         arxiv: 1903.04750
+        github: https://github.com/wencolani/CrossE
```

Comment:
It would be nice if we could render those in the documentation, too.

Reply:
we can make a pykeen flavor of parse_docdata that goes in and appends the docstring with a table containing citation information